src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -1390,6 +1390,9 @@ class Trial(Model):
   # Additional arguments to apply if selected.
   app_args = ndb.TextProperty()
 
+  # Flags that contradict the app args (separated by spaces).
+  contradicts = ndb.TextProperty(default='')
```

Comment:
Should we make this a `repeated=True` property instead?

Reply:
done